### PR TITLE
DIP-158 update names

### PR DESCRIPTION
**Pull request by YinonFirstDAG**
DIP-158 update names and removed duplicate abort sequence

### CODE REVIEW

File: dips/dip-158.md [hunk 1]
```diff
@@ -84,95 +85,86 @@ Some examples for such flows are:
...
 
 ## Command Types
 The following is a list of new values for the `command_type` field of the `CommandRequestObject`. Each command type denotes a step in the off-chain negotiations and specifies the data of the request and response 
 * `GetPaymentInfo` - Allows obtaining payment information based on reference id
-* `InitChargePayment` - Initializes a payment process of type `charge`. Triggering this command by the wallet means the customer approved to proceed with the payment and that the wallet is satisified with the business infromation provided. Successful response from the receiver means that the receiver is satisfied with the payer data provided by the wallet 
+* `InitChargePayment` - Initializes a payment process of type `charge`. Triggering this command by the Wallet (Customer VASP) (Customer VASP) means the customer approved to proceed with the payment and that the Wallet (Customer VASP) (Customer VASP) is satisified with the business infromation provided. Successful response from the Merchant VASP means that the Merchant VASP is satisfied with the payer data provided by the Wallet (Customer VASP) (Customer VASP)
```
Comment (davidiw) [inline dips/dip-158.md:94]: Is there an error here? Double parentheses

File: dips/dip-158.md [hunk 2]
```diff
@@ -84,95 +85,86 @@ Some examples for such flows are:
...
-|[InitChargePayment](#initchargepayment)|Sender > Receiver|This functionality allows the wallet to init the payment process based on the payment details. This is likely to occur after the customer approved the payment request and the wallet verified the business data|Reference ID ; Payer Data|(optional) Recipient Signature|
-|[AbortPayment](#abortpayment)|Sender > Receiver / Receiver > Sender|This functionality allows both parties to abort the payment outside the standard flow of the protocol|Reference ID||
+|[GetPaymentInfo](#getpaymentinfo)|Customer VASP > Merchant VASP|By providing the object reference id the Wallet (Customer VASP) (Customer VASP) can retrieve the object information from the receiving VASP. In most cases, this would be the first command sent by the Wallet (Customer VASP) (Customer VASP). The Wallet (Customer VASP) is not expected to provide any payer data at this stage|Reference ID|Payment Details ; Business Data|
```
Comment (davidiw) [inline dips/dip-158.md:99]: same here (Customer VASP) (Customer VASP)

File: dips/dip-158.md [hunk 3]
```diff
@@ -594,17 +586,17 @@ For example, for the following values:
 |vasp_address|dm1pgxah7pxhzljvp3p4m9g0m3tm0qqqqqqqqqqqqqqgyftgh| 
 |reference_id|ad8d888a-1791-4b63-98e5-6f1d6ddb4411|
 
-The deeplink represented by QR code would be (the domain and path are examples - the real wallet domain/path should be used):
+The deeplink represented by QR code would be (the domain and path are examples - the real Wallet (Customer VASP) domain/path should be used):
 
-`diem://some-diem-wallet.com/pay?vasp_address=dm1pgxah7pxhzljvp3p4m9g0m3tm0qqqqqqqqqqqqqqgyftgh&reference_id=ad8d888a-1791-4b63-98e5-6f1d6ddb4411`
+`diem://some-diem-Wallet (Customer VASP).com/pay?vasp_address=dm1pgxah7pxhzljvp3p4m9g0m3tm0qqqqqqqqqqqqqqgyftgh&reference_id=ad8d888a-1791-4b63-98e5-6f1d6ddb4411`
```
Comment (davidiw) [inline dips/dip-158.md:591]: fix

**Review by davidiw — COMMENTED** [review]
looks like you did some sort of search and replace, it didn't apply very cleanly (Customer VASP) (Customer VASP)

File: dips/dip-158.md (hunk 1 above)
Comment (YinonFirstDAG) [reply dips/dip-158.md]: Indeed find/replace error.
fixed.

File: dips/dip-158.md (hunk 3 above)
Comment (YinonFirstDAG) [reply dips/dip-158.md]: Fixed

File: dips/dip-158.md (hunk 2 above)
Comment (YinonFirstDAG) [reply dips/dip-158.md]: Fixed

**Review by dimroc — APPROVED** [review]

**Review by davidiw — APPROVED** [review]

**Pull request merged**